src/module/TaskManagement/config/module.config.php

```diff
@@ -60,5 +57,69 @@
 				'TaskManagement' => __DIR__ . '/../public',
 			),
 		),
-	),
-);
\ No newline at end of file
+    ),
+    
+    'bjyauthorize'=> array(
+    
+    	'resource_providers' => array(
+            'BjyAuthorize\Provider\Resource\Config' => array(                
+				'Ora\ReadModel\Task' => array(),
+    			'DoctrineORMModule\Proxy\__CG__\Ora\ReadModel\Stream' => array(),
```

Comment:
rigo 47 di TaskJsonModel: if ($this->authorize->isAllowed($r->getStream(), 'TaskManagement.Task.create'))
Ho provato infatti ad usare la classe Ora\ReadModel\Stream come risorsa, ma l'asserzione non viene richiamata e quindi isAllowed restituisce false.